### PR TITLE
i18n(id): Update css-and-tailwind.mdx translation

### PR DESCRIPTION
**Pull request by iyyafi**
#### Description
- Update ID translation guides for css-and-tailwind.mdx file

### CODE REVIEW

File: docs/src/content/docs/id/guides/css-and-tailwind.mdx [hunk 1]
```diff
@@ -22,7 +26,7 @@ Sesuaikan style yang diterapkan pada website Starlight Anda dengan memberikan fi
 
 2. Tambahkan path file CSS Anda ke dalam array `customCss` pada config Starlight di `astro.config.mjs`.
 
-   ```js
+   ```diff lang="js"
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/css-and-tailwind.mdx:29]: It looks like this is missing the [`+` diff indicators on 2 lines of code](https://github.com/withastro/starlight/pull/1102/files#diff-6429e29a42a27836094af0ffa6527c90616b826a953a84fa2e615ffd04cf64e5R35-R36) in this code block.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left one tiny comment to address some missing highlighs in a code block but otherwise, this looks great to me.

Note: I'm adding the `hacktoberfest-accepted` label right now, thanks for participating.

File: docs/src/content/docs/id/guides/css-and-tailwind.mdx (hunk 1 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/css-and-tailwind.mdx:29]: Ah i see, thank you for notice this. I will fix this soon.

**Review by HiDeoo — APPROVED** [review]
Thanks for updating the pull request, looks perfect to me now :rocket:

**Pull request merged**